editoast/src/converters/generate_routes.rs

```diff
@@ -401,16 +379,16 @@ mod tests {
     }
 
     #[test]
-    /* ----o---d>---
-            \------
+    /* ----o---<d>---
+            \-------
         The test case has one switch and one detector
     */
     fn generate_routes() {
         let railjson =
             crate::converters::osm_to_railjson::parse_osm("src/tests/routes.osm.pbf".into())
                 .unwrap();
         let routes = super::routes(&railjson);
-        assert_eq!(5, routes.len());
+        assert_eq!(6, routes.len());
```

Comment:
Did you analyze why this changed? We should ensure that we didn't break the import.

Reply:
"src/tests/routes.osm.pbf" provides infrastructure as shown in the image, so now we have 6 "routes" instead of the previous 5 due to "applicable_directions."
![g1996](https://github.com/osrd-project/osrd/assets/40186152/912a45f9-da65-4c7c-9bd6-a3c3df40c417)

Reply:
I see, but we might want to adapt the algorithm to take signal direction into account. It can be done in another PR.